l10n_ch_base_bank/tests/test_bank.py

```diff
@@ -173,7 +181,7 @@ def test_iban_postal_lower_case(self):
         self.assertEqual(account.acc_type, "iban")
 
     def test_other_bank(self):
-        self.bank.l10n_ch_postal = False
+        # self.bank.l10n_ch_postal = False
```

Comment:
You can remove this line.

This is an artefact of the former setUp function which was setting `ccp` on `self.bank`

https://github.com/OCA/l10n-switzerland/commit/118c9b2a71fb758dff2d7ab0c17f937420bd3199#diff-43f9985cce610ea14560cfe3362aa6162fbe069cb87560713657b3ae0758c170R22